src/server/heat_automation.py

```diff
@@ -208,11 +208,13 @@ def run_auto_frequency(self, heat_or_id, current_frequencies, num_nodes, calc_fn
                             if slot_match['slot'] != m_slot:
                                 eliminated_matches.append(slot_match)
                         del available_seats[an_idx]
+                        # keep only seats that are not close the the ones we choose
+                        available_seats = [x for x in available_seats if x['frq']['f'] not in range(m_node['frq']['f'] - 10, m_node['frq']['f'] + 10)]
```

Comment:
This is the way I use to remove available seats that are close the the one affected in that loop.